cmd/livepeer/livepeer.go

```diff
@@ -101,7 +101,9 @@ func main() {
 		if *monitor && *monhost == "" {
 			*monhost = "http://metrics-rinkeby.livepeer.org/api/events"
 		}
+		*datadir = *datadir + "/rinkeby"
 	} else if *devenv {
+		*datadir = *datadir + "/dev"
 	} else {
 		if !*offchain {
```

Comment:
Perhaps also have an offchain folder so it's separate from mainnet?

Reply:
@j0sh well, it seems the check for `offchain` is nested within both `rinkeby` and `mainnet` checks, so do you think the folders should be `mainnet_offchain` and `rinkeby_offchain` beside the existing `mainnet` and `rinkeby` onchain folders? 

P.s. Sorry about the late reply. the notification slipped through 🙏 😄

Reply:
Just one `offchain` for both, since there isn't really a concept of "rinkeby offchain" vs "mainnet offchain" when there's supposed to be no chain at all. That bit of code is quite ugly, hence https://github.com/livepeer/go-livepeer/issues/580